substrate/client/informant/src/lib.rs

```diff
@@ -39,7 +39,7 @@ fn interval(duration: Duration) -> impl Stream<Item = ()> + Unpin {
 /// The format to print telemetry output in.
```

Comment:
Probably this doc should be changed.
By definition telemetry is collection of remote information, but here we are displaying info coming from local node.